apps/site/lib/site_web/templates/content/_content_list.html.eex

```diff
@@ -6,3 +6,11 @@
       teasers: @content.teasers,
       type: Keyword.get(@content.recipe, :type, :generic),
       conn: @conn) %>
+
+<%= if list_cta?({@content.ingredients.type, @content.cta}, @content.teasers, @conn.path_info) do %>
```

Comment:
@amaisano it looks like we're putting `@content.ingredients.type` and `@content.cta` into a tuple here, only to just pattern-match it back into two separate params in `list_cta`. Unless there's some reason to keep the tuple I'm not seeing, how about just passing them separately?